app/models/manager_refresh/inventory_collection/builder/cloud_manager.rb

```diff
@@ -77,99 +103,110 @@ def orchestration_templates
         end
 
         def orchestration_stack_ancestry
-          orchestration_stack_ancestry_save_block = lambda do |_ems, inventory_collection|
-            stacks_inventory_collection = inventory_collection.dependency_attributes[:orchestration_stacks].try(:first)
-
-            return if stacks_inventory_collection.blank?
-
-            stacks_parents = stacks_inventory_collection.data.each_with_object({}) do |x, obj|
-              parent_id = x.data[:parent].try(:load).try(:id)
-              obj[x.id] = parent_id if parent_id
-            end
-
-            model_class = stacks_inventory_collection.model_class
-
-            stacks_parents_indexed = model_class
-                                     .select(%i(id ancestry))
-                                     .where(:id => stacks_parents.values).find_each.index_by(&:id)
-
-            ActiveRecord::Base.transaction do
-              model_class.select(%i(id ancestry))
-                         .where(:id => stacks_parents.keys).find_each do |stack|
-                parent = stacks_parents_indexed[stacks_parents[stack.id]]
-                stack.update_attribute(:parent, parent)
-              end
-            end
-          end
+          @options[:auto_inventory_attributes] = false
```

Comment:
I am not much of fan of instance variable interface, could we have some setter that would set the options just for this 1 InventoryCollection? Otherwise there can be hard to find side effects